packages/bot-web-ui/src/components/bot-notification/bot-notification.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { toast } from 'react-toastify';
+import { notification_style, TAction, TNotificationContent, TNotificationStyle } from './bot-notification-utils';
+
+export const NotificationContent: React.FC<TNotificationContent> = ({ message, primary_action }) => {
+    return (
```

Comment:
should the content be passed from outside as children?

Reply:
Yes we could do that approach too. In this approach i passed message to keep it clear on what we are expecting here. And for the unifying i've created notification-utils that hold all the existing notification messages and in the future, any new messages can be easily added and kept in one place. @vinu-deriv

Reply:
one point I am not sure is if in future they decide to have action button between text as it is now, then how will we handle . or just to make a word bold in the sentence

Reply:
There was discussion that action button would only be either on the right or left. And if the the text is required to be styled differently or needs to have a link(unlikely since we have dedicated action button now) then we can use Localize component as we have been doing previously. @vinu-deriv